### ISSUE TITLE
'Background Request URL' make rule fails

### ISSUE THREAD

**Issue by Dimas-sc** [arunelias/session-alive#1]
I'm trying to configure the addon to keeping alive an OWA (Outlook Web App) session.

The 'Trigger URL' it's ok because Session Alive detects it and activates the rule. But when I configure 'https://ecorreu.gencat.cat/owa/keepalive.owa' as a 'Background Request URL' the rule isn't activated anymore, and if I remove it works again. Why? 'Background Request URL' should not affect the trigger URL isn't it?

Thx

**Comment by Dimas-sc** [comment]
With this URL the same occurs: 'https://ecorreu.gencat.cat/owa/auth/logon.aspx'.

Leaving 'Background Request URL' empty or setting it to 'https://ecorreu.gencat.cat/owa/#path=/mail' makes trigger works again, but then the session isn't keeped alive.

**Comment by arunelias** [comment]
On my OWA at https://outlook.office365.com/owa/?realm=office.com&path=/mail/inbox i didn't see requests to /owa/keepalive.owa. I can see a GET request to /owa/ping.owa?UA=0 and you can try setting this as a Background Request.

**Comment by Dimas-sc** [comment]
With ping.owa the rule doesn't work. I think it's a bug.

I configured two identic rules, with the same 'Trigger URL' but the first with 'Background Request URL' set to '(...)/owa/#path=/mail' and the second set to '(...)/owa/ping.owa?UA=0'.
The first is activated after login but not de second. The same happened with keepalive.owa.

**Comment by arunelias** [comment]
I miss to mention about the `Stop Background Request when server response is not 200 OK` option in the [background request options](https://github.com/arunelias/session-alive#background-request-options)

I can see the response from the server for URL `/owa/ping.owa?UA=0` is 242 and not 200 OK. You need to remove check-mark on that option, and the add-on will continue to request the url. 

However the add-on will show a notification `Background Request is stopped because server response is not 200` if rule is stopped due to that, and is set by default at the [notification options](https://github.com/arunelias/session-alive#notification-options)

**Comment by Dimas-sc** [comment]
The problem is that with `Background Request URL` set to `/owa/ping.owa?UA=0` the rule is not even triggered. Changing it to `/owa/#path=/mail` makes it trigger ok, but it doesn't work as it doesn't renew the session. I tried disabling the `Stop Bacgkround Request... 200 OK` but it didn't fix the problem. Is like some `Background Request URL` brakes the addon.

If I can do something to debug tell me please.

**Comment by arunelias** [comment]
Fixed in version 1.1 Please update.

**Comment by arunelias** [comment]
Yes. If same trigger url is used in two rules then only one is activated. Duplicate checking is yet to be implemented. You can change the trigger url of 1st one to some dummy url and check. For debugging you can open firefox browser toolbox (Ctrl + Shift + Alt + I) and look in the console for message like below.

![browser_toolbox](https://user-images.githubusercontent.com/32271975/46649496-e6b66680-cbb6-11e8-9f86-4dc95a164ccb.jpg)

**Comment by Dimas-sc** [comment]
Yes, it works!!! Thx 👍
